build/bin/build-server-rpm

```diff
@@ -254,6 +254,7 @@ build() {
                 rpmbuild --define '_rpmdir %{_topdir}/RPMS/${spec_name}-${rpm_version}' --define 'dist .el7' -ba SOURCES/${spec_name}.spec
                 rm -f SOURCES/${spec_name}.spec*
             else
+                export PATH=/usr/local/go/bin/:$PATH
```

Comment:
@Sevlin Can we move that line and a few below to the spec file itself? That huge if/else code should become smaller over time, not larger.

Reply:
I think, the better way is to use alternatives, as we do for Grafana. Hence, we won't need to update `PATH` in runtime, build and dev envs.
```
sudo update-alternatives --install \"/usr/bin/go\" \"go\" \"/usr/local/go/bin/go\" 0
sudo update-alternatives --set go /usr/local/go/bin/go
```

Reply:
$PATH variable hasn't changed after these commands

`sudo echo $PATH
/home/adivinho/golang/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin

sudo update-alternatives --install /usr/bin/go go /usr/local/go/bin/go 0
sudo update-alternatives --set go /usr/local/go/bin/go

sudo echo $PATH /home/adivinho/golang/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin`

Reply:
@adivinho I've added alternative for `gofmt` in the Dockerfile